pandas/_libs/tslibs/fields.pyx

```diff
@@ -4,6 +4,7 @@ objects and arrays
 """
 from locale import LC_TIME
 
+from _strptime import LocaleTime
```

Comment:
1) the upstream version looks really stable, https://github.com/python/cpython/commits/main/Lib/_strptime.py last commit 13 months ago (looks like we didn't upstream), then 14 months before that, then almost two years...

2) pandas memory footprint at import-time has increased by 8mb since 1.2.0, largely driven by having more cython.  I think its worth trying to economize on this front.  This cuts 385kb.

3) I don't think we're getting any real perf benefit from this being in cython